### ISSUE TITLE
Update sdl2_ttf.pas to match code style guidelines

### ISSUE THREAD

**Issue by Free-Pascal-meets-SDL-Website** [PascalGameDevelopment/SDL2-for-Pascal#59]
The function declarations have to be checked for the` cdecl` modifier to satisfy our [code style guidlines](https://github.com/PascalGameDevelopment/SDL2-for-Pascal/blob/master/CHEATSHEET.md) (at the bottom). The `cdecl` modifier should be behind the `function` line, the `external` modifier should be in the next line with two letters indentention.

Best regards

**Comment by bunnylin** [comment]
Lazarus comes with a Jedi formatter. I haven't tried to use it, but at least its settings dialog has boxes for preferring tabs over spaces. Could be the best available Pascal formatter? Certainly more maintained than ptop.

**Comment by Free-Pascal-meets-SDL-Website** [comment]
> So far, we've been formatting code manually, which led to a lot of spent time and many inconsistencies. Perhaps we should consider using some automatic code formatter?

Good idea.

> e.g. I don't see a way to indent by tabs, instead of spaces, nor even to configure how many spaces should be used for indentation

Actually both should be possible ([https://wiki.freepascal.org/PTop](https://wiki.freepascal.org/PTop)).
The option for tab indentation is `inbytab`. - Are tabs preferably to two-space indententions, btw.?
For adding spaces one could use `spbef` as often as many spaces are needed, I would guess.

> Lazarus comes with a Jedi formatter.

Had no time to look into it, yet.

**Comment by suve** [comment]
> The option for tab indentation is `inbytab`.

Nope. I played around with `ptop` and that just tells the program to insert an indent. 

> Lazarus comes with a Jedi formatter.

Using something built into Lazarus would be problematic from a CI standpoint; we'd most likely want to include a "is the code formatted correctly?" check in the pipeline.

I took a brief look into Lazarus sources and the Jedi Formatter can be built as a stand-alone app, so we could do just that. But in thas case, it would probably be a good idea to create a container image with `fpc` and `sdl2` installed and the formatter already built, instead of doing that from scratch on each CI run.
